porcupine/plugins/run/no_terminal.py

```diff
@@ -12,7 +12,8 @@
 from pathlib import Path
 from typing import Callable
 
-from porcupine import get_tab_manager, images, textutils, utils
+from porcupine import get_tab_manager, images, utils
```

Comment:
textutils needed for `LinkManager`
```suggestion
from porcupine import get_tab_manager, images, textutils, utils
```